test/e2e_node/device_plugin.go

```diff
@@ -53,7 +53,7 @@ var _ = framework.KubeDescribe("Device Plugin [Feature:DevicePlugin][NodeFeature
 
 var _ = framework.KubeDescribe("Device Plugin [Feature:DevicePluginProbe][NodeFeature:DevicePluginProbe][Serial]", func() {
 	f := framework.NewDefaultFramework("device-plugin-errors")
-	testDevicePlugin(f, true, "/var/lib/kubelet/plugins_registry")
+	testDevicePlugin(f, true, "/var/lib/kubelet/plugins_registry/")
```

Comment:
The other place which call testDevicePlugin passes DevicePluginPath and that is "/var/lib/kubelet/device-plugins/".
So lgtm

Reply:
@yujuhong did you mean making `pluginSockDir` as a local variable in function `testDevicePlugin`, instead passing it as an input argument? If yes, we will have to use `if-else` to initialize `pluginSockDir` to correct path depending on value of `enablePluginWatcher`. Is that what you are suggesting?

Reply:
```
func testDevicePlugin(f *framework.Framework, enablePluginWatcher bool, pluginSockDir string)
```
The name  "pluginSockDir" already assumes a path to the directory. It might make sense to just use `filepath` to join the paths, instead of always making sure the path ends with `/'